### ISSUE TITLE
[feature request] Mac Touch Bar tabs navigation

### ISSUE THREAD

**Issue by CoVoCre** [brave/brave-browser#5076]
As requested for chromium https://bugs.chromium.org/p/chromium/issues/detail?id=778013&hotlist_id=5607&sort=%20rank%20-ID , it'd be amazing to have safari-like tab navigation from the touch bar on macs.
Chromium edge on mac has it now ! I don't know why they didn't add this to chromium as they seem to say they want to contribute, but hey, here is the idea for Brave :)
Here's a screenshot of chromium edge's tab in the touchbar controls : 
<img width="1085" alt="Capture d’écran Touch Bar 2019-06-30 à 00 24 33" src="https://user-images.githubusercontent.com/25605953/60390021-7e553f80-9acd-11e9-93ba-2b02ce3fadff.png">

And one of how safari does it (it renders a preview of the top of the pages for each tab) : 
<img width="1085" alt="Capture d’écran Touch Bar 2019-06-30 à 00 25 54" src="https://user-images.githubusercontent.com/25605953/60390031-ad6bb100-9acd-11e9-9759-ae56522db044.png">

Oh, and because I'm having fun, I see Opera's supposed to have it so I installed it and there you go (it seems to me like the most elegant implementation of the three) : 
<img width="1085" alt="Capture d’écran Touch Bar 2019-06-30 à 00 28 46" src="https://user-images.githubusercontent.com/25605953/60390055-13f0cf00-9ace-11e9-9bea-7a5d6443c1ac.png">

**Comment by cburns2017** [comment]
Hoping this issue gets picked up. Would love to see more support for customizing the touchbar

**Comment by syddo** [comment]
> Hoping this issue gets picked up. Would love to see more support for customizing the touchbar

I agree... I've been using Brave on Windows and now I'm moving to a Mac I keep using Safari because of the Touch Bar features. Is this issue being worked on?

**Comment by lubaroli** [comment]
+1 for this request.

**Comment by gokayyildiz** [comment]
Are there any updates for this issue?

**Comment by nsklaus** [comment]
would it be possible that current brave is already having unfinished code that is somehow wrecking havoc on macos touchbar ?

maybe it's just me being a bit paranoid, and to be clear: i have nothing to back this up, 
but i was having issues with my touchbar, i thought it was dying, it was blinking, displaying garbage..
in an attempt to diagnose the problem, i reinstalled a barebone macos install with only apple software.
and touchbar was working correctly again. after a while i tried to use brave browser again instead of plain safari, and the touchbar problems started to be back.
maybe it's just a coincidence or anyone else can confirm this ?
